src/test/kotlin/no/nav/syfo/personstatus/api/v2/PersontildelingApiV2Spek.kt

```diff
@@ -34,7 +34,7 @@ object PersontildelingApiV2Spek : Spek({
 
             val baseUrl = personTildelingApiV2Path
 
-            afterEachTest {
+            beforeEachTest {
```

Comment:
Blir følgefeil når vi bruker `beforeEach` og `afterEach` i to påfølgende filer 🤷🏼‍♂️ Må være consistent